be/tool/beck.c

```diff
@@ -515,6 +524,10 @@ int main(int argc, char **argv)
 	result = scanner_init(&s);
 	if (result != 0)
 		err(EX_CONFIG, "Cannot initialise scanner.");
+	if(dry_run) {
```

Comment:
add a space after "if"

Reply:
If not dry_run, then the SIGINT is not called. So CTRL+C will directly terminate this program.
Is this the expected behavior?

Reply:
In case of dry run now we are looking for printing summary of records scanned till 'Ctrl+C' is pressed.

Reply:
Yes. I know in case of dry run, "CTRL+C" will set the flag "sig_caught" and then exit the loop, and then print out summary.

But in NON dry run (real run), the "CTRL+C" signal is not handled, and that will terminate the program run abruptly.

Reply:
1. In case of dry run the beck is invoked directly from command line, hence can receive signal. Also the dry run is read only mode and graceful termination is simpler.
2. In case of normal run the beck tool is invoked via recovery script and my not receive signal, also the normal run modifies data hence graceful shutdown will required additional logic. So the existing behavior is kept as is for normal run.

Reply:
> add a space after "if"

done.

Reply:
As discussed in call.  We will be checking with Nikita and can be taken as PR.